src/web3/createMulticallContractInstance.ts

```diff
@@ -0,0 +1,35 @@
+import MulticallAtrifact from "@uma/core/build/contracts/MulticallMock.json";
+import { ethers } from "ethers";
+
+// interface Network {
+//   [key: string]: {
+//     address: string;
+//     events: object;
+//     links: object;
+//     transactionHash: string;
+//   };
+// }
```

Comment:
Commented out?

Reply:
Only used if the networks object is defined. I wasn't sure if this would be the case, but it's looking like we aren't going to be using artifacts anyway.